### DISCUSSION TITLE
【建议】想认真写个建议贴，开发者看看这里！

### DISCUSSION THREAD

**Discussion by travellernotfound** [share121/inter-knot#1860] [其他]
![Uploading 《原神》PV短片——「诸苦无隙」壁纸截图5.jpg…]()

有一个小想法，开发者可以创建一个**置顶的建议帖**，大家集思广益来为这个网站的建设添砖加瓦，提供更好的建设想法。比如建立一个更方便的举报机制，设立一个完善的社区规则，开发点赞收藏这样的功能等等。 也可以有效防止一些有心人故意传播淫秽色情或者发表不当言论。
这个网站很有意思，感觉是一个很高质量的二创了，而且相信很多绳网用户也会希望这个网站变得更加完善，所以还是希望开发者团队可以看到这个帖子。 时间很多，可以慢慢完善这个网站。

**Comment by travellernotfound** [comment]
顶一下

**Comment by travellernotfound** [comment]
111

**Comment by ZhuYuan-148** [comment]
discord可以提建议的

**Comment by dongzhiaidao** [comment]
+1
